src/main/java/laura/Date.java

```diff
@@ -0,0 +1,44 @@
+package laura;
+
+import laura.exception.LauraException;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class that deals with formatting of dates
+ */
+public class Date {
+    private final LocalDate date;
+    /** The format that the date should be written in string */
+    private static final DateTimeFormatter stringFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
I noticed that this could be a constant, perhaps use screaming camel case instead?
```suggestion
    private static final DateTimeFormatter STRING_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```